src/components/LandingPage/index.js

```diff
@@ -11,88 +11,88 @@ import { ReactComponent as Container } from '../../assets/images/container.svg';
 import { ReactComponent as Orch } from '../../assets/images/orcherstration.svg';
 import { ReactComponent as Registry } from '../../assets/images/registry.svg';
 import { ReactComponent as Cloud } from '../../assets/images/multicloud.svg';
-import './LandingPage.css';
+import Styles from './LandingPage.module.css';
 
 const LandingPage = (props) => {
   let { user } = props;
   if (user.data.id === undefined) {
     user = false;
   }
   return (
-    <div className="LandingPageMain">
+    <div className={Styles.LandingPageMain}>
```

Comment:
HEy @neelxie , this is out of curiosity, Why are you using this kind of  `CSS` class call?

Reply:
@Mubangizi it's because just like the PR section "What does this PR do?" -- It mentions that we are introducing CSS modules as a counter against class name issues.